apps/os/backend/db/schema.ts

```diff
@@ -45,9 +45,12 @@ export const session = pgTable("session", (t) => ({
   token: t.text().notNull().unique(),
   ipAddress: t.text(),
   userAgent: t.text(),
-  userId: t.text().notNull(),
+  userId: t
+    .text()
+    .notNull()
+    .references(() => user.id, { onDelete: "cascade" }),
   // https://www.better-auth.com/docs/plugins/admin#schema
-  impersonatedBy: t.text(),
+  impersonatedBy: t.text().references(() => user.id, { onDelete: "set null" }),
```

Comment:
imo this should be cascade too. we don't want a session hanging around when the admin impersonating gets deleted. that'd make the session look like a normal one